docker/bin/activate

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Can we add this instead:

#!/bin/bash
# strict mode http://redsymbol.net/articles/unofficial-bash-strict-mode/
set -euo pipefail
IFS=$'\n\t'

It has saved me a ton of headache and all of the other bin files has it.